### PR TITLE
Implemented assignment-1

### PR DESCRIPTION
**Pull request by sohial19**
In visual studio we created single line arguments.

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+public class Main{
```
Comment (raaaimund) [inline Main.java]: Please only commit formatted code!

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    public static void main(String[] args){
```
Comment (raaaimund) [inline Main.java]: Please only commit formatted code!

File: Main.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        if(args.length>0){
```
Comment (raaaimund) [inline Main.java]: Please only commit formatted code!

File: Main.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        for(String val: args)
```
Comment (raaaimund) [inline Main.java]: Please only commit formatted code!

File: Main.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+        System.out.println(val);
```
Comment (raaaimund) [inline Main.java]: Check the assignment's description again, only the FIRST argument should be printed. Not all arguments!

File: Main.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    }
+        else{
+
+            System.out.println("There is no arguments are provided");
+
+        
+    }
+
+    }
+}
```
Comment (raaaimund) [inline Main.java:9]: At the end of the file, there should be a newline.

File: Main.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
+public class Main{
+    public static void main(String[] args){
+        if(args.length>0){
+        for(String val: args)
+        System.out.println(val);
+    }
+        else{
+
```
Comment (raaaimund) [inline Main.java]: Please only commit formatted code!

File: Main.java [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+            System.out.println("There is no arguments are provided");
+
+        
+    }
```
Comment (raaaimund) [inline Main.java]: Please only commit formatted code!

File: Main.java [hunk 9]
```diff
@@ -0,0 +1,15 @@
+public class Main{
+    public static void main(String[] args){
+        if(args.length>0){
+        for(String val: args)
+        System.out.println(val);
+    }
+        else{
```
Comment (raaaimund) [inline Main.java]: Please only commit formatted code!

File: Main.java (hunk 8 above)
Comment (uday667) [reply Main.java]: yeah! we understood what you said. I will push same !!

**Comment by raaaimund** [comment]
.class files (e.g. Main.class) should not be committed! Use a .gitignore file to ensure that .class files won't be submitted in the future (https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/) and remove the current Main.class file.

**Comment by raaaimund** [comment]
If you do not want to implement the remaining comments, please merge the branch to be ready for the next assignment. If you implement the remaining comments I can give you extra points.

**Pull request merged**